src/main/drivers/serial_usb_vcp.c

```diff
@@ -99,7 +99,7 @@ static void usbVcpWriteBuf(serialPort_t *instance, const void *data, int count)
     uint32_t start = millis();
     const uint8_t *p = data;
     while (count > 0) {
-        uint32_t txed = CDC_Send_DATA(p, count);
+        uint32_t txed = CDC_Send_DATA(p, (count > 255) ? 255 : count);
```

Comment:
I did not want to touch the CDC_* stuff as this is an external library. Of course this looks like the better fix :)